File: courses_2023/js_101_2023/lesson_3/easy_3_1.js
/* Lesson 3: Practice Problems: Easy 3.1

Instructions:
- Write three different ways to remove all of the elements from the array.

Input:
- Array

Output:
- All elements are removed from input array

Rule:
- Solve the problem in 3 different ways
*/

let numbers1 = [1, 2, 3, 4];
let numbers2 = [1, 2, 3, 4];
let numbers3 = [1, 2, 3, 4];

numbers1.length = 0;

while (numbers2.length > 0) {
  numbers2.pop();
}

numbers3.splice(0);


//Test cases
console.log(numbers1); // []
console.log(numbers2); // []
console.log(numbers3); // []

File: python-track/py101/lesson_3/easy_3_1.py
# Write two different ways to remove all of the elements from the following list:

numbers1 = [1, 2, 3, 4]
numbers2 = [1, 2, 3, 4]
numbers3 = [1, 2, 3, 4]

for _ in range(len(numbers1)):
    numbers1.pop()

while numbers2:
    numbers2.pop()

numbers3.clear()

print(numbers1) # []
print(numbers2) # []
print(numbers3) # []